### PR TITLE
Added ability to provide multiple merchant capabilities

### PR DESCRIPTION
**Pull request by michaelgobbers**
I was missing the ability to add multiple merchant capabilities, I noticed this was raised before  #20  so I took a moment to add them. 

The example is now showing an example use of the merchantcapabilites parameter

```
AppleParameters(
    merchantIdentifier: 'example_id',
    billingContact: Contact(
      emailAddress: 'test@test.com',
      postalAddress: PostalAddress(
        street: 's',
        city: 'c',
        state: 'st',
        postalCode: '123321',
        country: 'ct',
      ),
      name: PersonNameComponents(
        familyName: 'qwe',
        middleName: 'ewq',
        namePrefix: 'a',
        nameSuffix: 'h',
        nickname: 'test',
        phoneticRepresentation: PersonNameComponents(
          middleName: 'ewq2',
          givenName: 'rty2',
          namePrefix: 'a2',
          nameSuffix: 'h2',
          nickname: 'test2',
        ),
      ),
    ),
    shippingContact: Contact(
      emailAddress: 'test@test.com',
    ),
    merchantCapabilities: <MerchantCapabilities>[
      MerchantCapabilities.threeds,
      MerchantCapabilities.credit,
    ],
);
```

### CODE REVIEW

File: packages/mad_pay_ios/ios/Classes/PaymentHelpers.swift [hunk 1]
```diff
@@ -97,19 +97,31 @@ class PaymentNetworkHelper {
...
-            return PKMerchantCapability.capability3DS
+    static func getMerchantCapabilities(_ merchantCapabilities: [Apple_MerchantCapabilities]?) -> PKMerchantCapability {
+
+        if(merchantCapabilities == nil){
+            return .capability3DS;
+        }
+
+        var decodedCapabilities: PKMerchantCapability = [];
+
+        if(merchantCapabilities!.contains(Apple_MerchantCapabilities.threeds) ){
```
Comment (mit-73) [inline packages/mad_pay_ios/ios/Classes/PaymentHelpers.swift:108]: Maybe we should add a guard check for everyone?

File: packages/mad_pay_platform_interface/lib/src/models/apple_parameters.dart [hunk 2]
```diff
@@ -54,7 +54,8 @@ extension AppleParametersX on AppleParameters {
   /// Mapped this model to proto model
   proto.AppleParameters get toProto => proto.AppleParameters(
         merchantIdentifier: merchantIdentifier,
-        merchantCapabilities: merchantCapabilities?.toProto,
+        merchantCapabilities:
+            merchantCapabilities?.map((MerchantCapabilities e) => e.toProto!),
```
Comment (mit-73) [inline packages/mad_pay_platform_interface/lib/src/models/apple_parameters.dart:58]: You can get rid of the bang operator here if you modify toProto:
```
/// MerchantCapabilities extension
extension MerchantCapabilitiesX on MerchantCapabilities {
  /// Mapped this model to proto model
  proto.MerchantCapabilities get toProto {
    switch (this) {
      case MerchantCapabilities.threeds:
        return proto.MerchantCapabilities.THREEDS;
      case MerchantCapabilities.credit:
        return proto.MerchantCapabilities.CREDIT;
      case MerchantCapabilities.debit:
        return proto.MerchantCapabilities.DEBIT;
      case MerchantCapabilities.emv:
        return proto.MerchantCapabilities.EMV;
    }
  }
}
```

You can also fix ShippingType:

```
/// ShippingType extension
extension ShippingTypeX on ShippingType {
  /// Mapped this model to proto model
  proto.ShippingType get toProto {
    switch (this) {
      case ShippingType.shipping:
        return proto.ShippingType.SHIPPING;
      case ShippingType.delivery:
        return proto.ShippingType.DELIVERY;
      case ShippingType.servicePickup:
        return proto.ShippingType.SERVICE_PICKUP;
      case ShippingType.storePickup:
        return proto.ShippingType.STORE_PICKUP;
    }
  }
}
```

**Comment by mit-73** [comment]
Thanks for your PR, I looked it up, could you fix it?

File: packages/mad_pay_ios/ios/Classes/PaymentHelpers.swift (hunk 1 above)
Comment (michaelgobbers) [reply packages/mad_pay_ios/ios/Classes/PaymentHelpers.swift:108]: I added some guard checks on merchantCapabilities being null

**Comment by michaelgobbers** [comment]
Made the changes you proposed. Could you take a look at it again?

**Comment by michaelgobbers** [comment]
I also bumped flutter_svg version as they recently released bumped the major version.

**Pull request merged**